.Rbuildignore

```diff
@@ -1,3 +1,4 @@
+^\.gitlab-ci\.yml$
```

Comment:
I don't think the usethis `.gitignore` file needs this. Is this perhaps leftover from some experimentation? If so, can you remove this change?

Reply:
Removed it in https://github.com/r-lib/usethis/pull/562/commits/4fe65038e08dab007556adb5602e5760926ad925